app/src/main/java/com/example/vybe/VibeEvent.java

```diff
@@ -0,0 +1,53 @@
+package com.example.vybe;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+public class VibeEvent implements Serializable {
+    private Vibe vibe;
+    private LocalDateTime dateTime;
+    private String reason;
+    private String socialSituation;
+
+    public VibeEvent(Vibe vibe, LocalDateTime dateTime, String reason, String socialSituation) {
```

Comment:
Do we want an empty parameter constructor that gives these values predefined defaults?